src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -60,15 +60,17 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Core" Version="1.9.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.9.0-alpha.20210317.1" />
     <PackageReference Include="DotNetTI.BreakingChangeAnalysis" Version="1.1.0-preview" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.0" />
     <PackageReference Include="Microsoft.Azure.AppService.Middleware.Functions" Version="1.4.0" />
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.11020001-fabe022e" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.27" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="4.0.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="5.0.0" />
```

Comment:
Submitting this comment only as this is a foundational thing we need to follow up on before we can look at the other changes. We can't have a major version change in the host without impacting compatibility. I'll reach out so we can discuss offline, as I think this is being impacted by the layering upstream.